model/project_aliases.go

```diff
@@ -343,6 +346,32 @@ func CopyProjectAliases(oldProjectId, newProjectId string) error {
 	return nil
 }
 
+func FindParametersForPatchAlias(identifier string, v *Version) ([]patch.Parameter, error) {
```

Comment:
Rather than getting aliases from the DB and config each time we call FetchExpansions, could we instead cache them with the patch at the time that we finalize it? (This might allow us to just look at what aliases are being used at that time as well, although I'm not totally sure)

Reply:
Sure, I can move this logic into `FinalizePatch`